### PR TITLE
Changed Kindstore UI

### PR DESCRIPTION
**Pull request by Shikhar1903**

### CODE REVIEW

**Comment by PrarabdhGarg** [comment]
@Shikhar1903 please make sure to change the assignees column also

File: app/src/main/res/layout/adapter_kind_items.xml [hunk 1]
```diff
@@ -14,56 +14,78 @@
...
+            android:gravity="center"
+            android:text="Tokens"
+            android:textColor="@color/colorWhite"
+            android:textSize="22sp"
+            app:layout_constraintBottom_toBottomOf="@+id/stallImage"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintHorizontal_bias="0.0"
+            app:layout_constraintStart_toEndOf="@+id/price"
+            app:layout_constraintTop_toTopOf="@+id/stallImage" />
+
```
Comment (PrarabdhGarg) [inline app/src/main/res/layout/adapter_kind_items.xml]: Please remove this extra line

File: app/src/main/res/layout/adapter_kind_items.xml [hunk 2]
```diff
@@ -14,56 +14,78 @@
...
-            app:layout_constraintHorizontal_bias="0.6"
-            app:layout_constraintStart_toEndOf="@+id/ItemName"
-            app:layout_constraintTop_toTopOf="@+id/ItemName" />
+            android:textSize="22sp"
+            app:layout_constraintBottom_toBottomOf="@+id/stallImage"
+            app:layout_constraintStart_toStartOf="@+id/stallImage"
+            app:layout_constraintTop_toTopOf="@+id/stallImage" />
 
     <ImageView
             android:id="@+id/kImg"
```
Comment (PrarabdhGarg) [inline app/src/main/res/layout/adapter_kind_items.xml]: Also, change the name of this image to a more meaningful one

File: app/src/main/res/layout/adapter_kind_items.xml [hunk 3]
```diff
@@ -14,56 +14,78 @@
...
+            android:layout_marginTop="25dp"
+            android:layout_marginBottom="25dp"
+            app:layout_constraintBottom_toBottomOf="@+id/stallImage"
             app:layout_constraintStart_toStartOf="@+id/stallImage"
-            app:layout_constraintTop_toTopOf="@+id/ItemName"
+            app:layout_constraintTop_toTopOf="@+id/stallImage"
             app:srcCompat="@drawable/ic_card_giftcard_black_24dp" />
 
+    <TextView
+            android:id="@+id/textView31"
```
Comment (PrarabdhGarg) [inline app/src/main/res/layout/adapter_kind_items.xml]: Change the id of this as well. Try avoiding default ids for widgets

File: app/src/main/res/layout/adapter_kind_items.xml [hunk 4]
```diff
@@ -14,56 +14,78 @@
...
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="24dp"
+            android:layout_marginTop="52dp"
+            android:layout_marginBottom="13dp"
+            android:fontFamily="@font/shellypalmer_circular_std_book"
+            android:gravity="center"
+            android:text="Tokens"
```
Comment (PrarabdhGarg) [inline app/src/main/res/layout/adapter_kind_items.xml]: Shift this text to the strings resource file

**Review by PrarabdhGarg — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/adapter_kind_items.xml (hunk 1 above)
Comment (Shikhar1903) [reply app/src/main/res/layout/adapter_kind_items.xml]: Okay

File: app/src/main/res/layout/adapter_kind_items.xml (hunk 2 above)
Comment (Shikhar1903) [reply app/src/main/res/layout/adapter_kind_items.xml]: Okay

File: app/src/main/res/layout/adapter_kind_items.xml (hunk 4 above)
Comment (Shikhar1903) [reply app/src/main/res/layout/adapter_kind_items.xml]: I have made the suggested changes.

File: app/src/main/res/layout/adapter_kind_items.xml [hunk 5]
```diff
@@ -9,61 +9,81 @@
...
+            app:layout_constraintTop_toTopOf="@+id/stallImage" />
 
     <ImageView
-            android:id="@+id/kImg"
-            android:layout_width="56dp"
-            android:layout_height="50dp"
-            app:layout_constraintBottom_toBottomOf="@+id/ItemName"
-            app:layout_constraintEnd_toStartOf="@+id/ItemName"
-            app:layout_constraintHorizontal_bias="0.28"
+            android:id="@+id/Stall_photo"
```
Comment (PrarabdhGarg) [inline app/src/main/res/layout/adapter_kind_items.xml]: According to the convention followed throughout the codebase, widget ids should start with lower case

File: app/src/main/res/layout/adapter_kind_items.xml [hunk 6]
```diff
@@ -9,61 +9,81 @@
...
+            android:layout_marginBottom="25dp"
+            app:layout_constraintBottom_toBottomOf="@+id/stallImage"
             app:layout_constraintStart_toStartOf="@+id/stallImage"
-            app:layout_constraintTop_toTopOf="@+id/ItemName"
+            app:layout_constraintTop_toTopOf="@+id/stallImage"
             app:srcCompat="@drawable/ic_card_giftcard_black_24dp" />
 
-</androidx.constraintlayout.widget.ConstraintLayout>
+    <TextView
+            android:id="@+id/Token_Text"
```
Comment (PrarabdhGarg) [inline app/src/main/res/layout/adapter_kind_items.xml]: Same as above message.

**Review by PrarabdhGarg — CHANGES_REQUESTED** [review]

**Comment by PrarabdhGarg** [comment]
@Shikhar1903 make sure to resolve the conversations for the issues you have already solved

**Review by PrarabdhGarg — APPROVED** [review]

**Pull request merged**